src/main/java/seedu/address/model/event/DayDeadline.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.event;
+
+import java.util.ArrayList;
+
+import seedu.address.model.event.exceptions.EmptyEventException;
+import seedu.address.model.event.exceptions.EventNotFoundException;
+
+public class DayDeadline {
+    private final ArrayList<Deadline> dailyDeadline;
+
+    public DayDeadline() {
+        this.dailyDeadline = new ArrayList<Deadline>();
+    }
+
+    public DayDeadline(ArrayList<Deadline> dailyDeadline) {
+        this.dailyDeadline = dailyDeadline;
+    }
+
+    public String addDeadline(Deadline deadline) {
+        this.dailyDeadline.add(deadline);
+        return "Deadline added.";
+    }
+
+    public String deleteDeadline(Deadline deadline) {
+        String deadlineDescription = deadline.getDescription();
+
+        boolean isRemoved = this.dailyDeadline
+                .removeIf(currDeadline -> (currDeadline.getDescription().equals(deadlineDescription)));
+        if (!isRemoved) {
+            throw new EventNotFoundException();
+        }
+        return deadline.toString() + " is removed.";
```

Comment:
Think we should standardize to either use `String.format` or string concatenation